boards/arm/mimxrt595_evk/mimxrt595_evk_cm33_defconfig

```diff
@@ -12,6 +12,7 @@ CONFIG_UART_CONSOLE=y
 CONFIG_SERIAL=y
 CONFIG_PINCTRL=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_DISK_DRIVER_SDMMC=y
```

Comment:
Do you want to always enable the SDMMC driver for this board? Shouldn't it default to enabled when disk drivers used?

Reply:
thanks for catch, it's not needed anymore due to this [commit on the main branch](https://github.com/zephyrproject-rtos/zephyr/commit/3b2456dd6e786605c697a4cad53efb067930709e)